crm_lead_vat/models.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class Lead(models.Model):
+    _inherit = "crm.lead"
+
+    vat = fields.Char(
+        "TIN",
+        help="Tax Identification Number. The first 2 characters are the "
+             "country code.")
+
+    @api.model
+    def _lead_create_contact(self, lead, name, is_company, parent_id=False):
+        """Add VAT to partner."""
+        return (super(Lead, self.with_context(default_vat=lead.vat))
+                ._lead_create_contact(lead, name, is_company, parent_id))
```

Comment:
[This is the line that fails in Travis](https://travis-ci.org/OCA/crm/jobs/86186745#L308), but as you can see this [is almost a copy of this other](https://github.com/OCA/l10n-spain/pull/224/files#diff-e5d75ea06e4728330dc5997a57171873R18), which is in another OCA repo and Travis is OK. Local tests work too.